scripts/publish.sh

```diff
@@ -87,6 +81,10 @@ npm version $VERSION
 NEW_VERSION=$(jq -r ".version" package.json)
 echo "Made a $VERSION version."
 
+echo "Preparing VSCode extension..."
+./scripts/publish-vscode.sh patch
```

Comment:
Hardcoding this to patch right now - open to switching this to match the CLI version type (ie if we release a minor version of the CLI, the accompanying VSCE release will be minor) - lmk what you prefer.

Reply:
I don't think it should match. VSCode features upgrades will definitely differ.

What does the CLI do? Can we specify the update "level" through changelog?